src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -232,8 +233,26 @@ protected function devRefreshDrupal(string $siteName = 'default'): Result
      */
     protected function landoUri($siteDir): string
     {
-        $landoConfigPath = "$this->drupalRoot/../.lando.yml";
-        $landoCfg = Yaml::parseFile($landoConfigPath);
+        $landoCfg = false;
+        $possibleLandoConfigPaths = [
+            // The usual suspect.
+            "$this->drupalRoot/../.lando.yml",
+            // The site could have a front-end and back-end site in different
+            // subdirectories with the lando.yml in the root directory.
+            "$this->drupalRoot/../../.lando.yml",
+        ];
+        foreach ($possibleLandoConfigPaths as $landoConfigPath) {
+            try {
+                $landoCfg = Yaml::parseFile($landoConfigPath);
+            } catch (ParseException $exception) {
+                $this->say("Unable to load Lando config from $landoConfigPath.");
+            }
+            // Break out of the loop if valid configuration was found.
+            if ($landoCfg === true) {
+                $this->say("Found Lando config at $landoConfigPath.");
+                break;
+            }
+        }
         // First, check for multisite proxy configuration.
```

Comment:
```suggestion
        if (!$landoCfg) {
             // Abort the process and let the user know what is up.
        }
        // First, check for multisite proxy configuration.
```

Reply:
Liked it, did it my way (hit song guaranteed)